pkg/progressbar/progressbar.go

```diff
@@ -208,62 +205,61 @@ func (p *ProgressBar) renderProgressBar() error {
 		return fmt.Errorf("failed to clear progress bar: %v", err)
 	}
 
-	// The progress bar starts with the description.
-	if _, err := builder.WriteString(p.description); err != nil {
-		return fmt.Errorf("failed to add description: %v", err)
+	// Emoji to indicate progress action (📥 for uploading).
+	emoji := color.New(color.FgYellow).Sprintf("📥  ")
+	if p.currentUploadPercentage == 100 {
+		emoji = color.New(color.FgGreen).Sprintf("✔  ")
```

Comment:
You can keep the same emoji (since we have info written below) but make it green, or delete this.